eng/pipelines/templates/stages/archetype-cpp-release.yml

```
@@ -86,53 +86,69 @@ stages:
                     steps:
                       - checkout: self
 
+                      - pwsh: |
+                          $branchName = "azure-sdk-for-cpp-${{ artifact.Name }}-$(Build.BuildId)"
+                          if ($env:VCPKGFORKBRANCHNAME) { 
```

nit: If we control this variable name, let's add underscores: `VCPKG_FORK_BRANCH_NAME`

```suggestion
                          if ($env:VCPKG_FORK_BRANCH_NAME) { 
```

---

Some more research reveals that we can use the macro syntax (`$(VcpkgForkBranchName)`) directly in the inline powershell script.